File: algoritmos-js/mdc.js
var n1 = parseInt(prompt("primeiro numero:  "));
var n2 = parseInt(prompt("segundo numero: "));

var menorNum, maiorNum;
var aux=0
if(n1>n2){
    maiorNum=n1;
    menorNum=n2;
}else{
    maiorNum=n2;
    menorNum=n1;
}
while((maiorNum%menorNum)!=0){
    aux=menorNum;
    menorNum=(maiorNum%menorNum);
    maiorNum=aux;
}
document.write("MDC de ("+ A + " , "+ B + ") = " + menorNum);

File: algoritmos-java/src/MDC.java
import java.util.Scanner;

public class MDC {
    public static void main(String[] args) {
		try(Scanner n = new Scanner(System.in)) {
			int num1=0;
			System.out.print("Digite o primeiro numero: ");
			num1 = n.nextInt();
			int num2=0;
			System.out.print("Digite o segundo numero: ");
			num2 = n.nextInt();
			
			int maior, menor;
			if(num1>num2) {
				maior= num1;
				menor= num2;
			} else {
				maior= num2;
				menor= num1;
			}
			int aux = 0;
			while((maior%menor)!=0) {
				aux= menor;
				menor= (maior%menor);
				aux= maior;
			}
			System.out.print("MDC de ("+ num1 + " , " + num2 + ") = " +
				    menor);
		}
  }
}